### PR TITLE
 added a new game tutorial.

### PR DESCRIPTION
**Pull request by sahninsh**
I have added a new game tutorial by freecodecamp which I believe would be very helpful to new programmers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@
 | 5.      | Checkers with AI      | https://www.youtube.com/playlist?list=PLzMcBGfZo4-myY28wdQuJDBi8pCt-GIj6 | <img src="https://img.shields.io/badge/-medium-orange">  |
 | 6.      | Hangman               | https://www.youtube.com/playlist?list=PLzMcBGfZo4-ndZlN21DasvpfKwIc1rI6w | <img src="https://img.shields.io/badge/-medium-orange">  |
 | 7.      | Flappy Bird with AI   | https://www.youtube.com/playlist?list=PLzMcBGfZo4-lwGZWXz5Qgta_YNX3_vLS2 | <img src="https://img.shields.io/badge/-medium-orange">  |
+| 8.      | Space invaders [Medium] | https://www.youtube.com/watch?v=FfWpgLFMI7w | <img src="https://img.shields.io/badge/-medium-orange">  |
```
Comment (Aayushi-Mittal) [inline README.md]: Instead of **[Medium]** write (Free Code Camp). It would be easy to differentiate.

Comment (sahninsh) [reply README.md]: Alright

Comment (sahninsh) [reply README.md]: I have fixed the issue .

**Review by Aayushi-Mittal — APPROVED** [review]
Great! Thanks for your contribution. 👍

**Comment by Aayushi-Mittal** [comment]
But please resolve the conflicts then I wil merge.

**Pull request merged**